File: back-end/src/main/java/com/spring/restaurant/model/Order.java
package com.spring.restaurant.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Order extends BaseEntity{

    private int price;
    private String img;
    private String description;



}

File: front-end/front-end/src/app/model/order.ts
export class Order {

    id: Number; 
    name: String;
    description: String;
    img: String;
    price: Number;
    data_create: Date;
    data_update: Date;


    constructor(id: Number, name: String, description: String, img: String, price: Number, data_create: Date, data_update: Date  ){
        this.id = id;
        this.name = name;
        this.description = description;
        this.img = img;
        this.price = price;
        this.data_create = data_create;
        this.data_update = data_update;
    }


}
